Automattic/Gravatar-SDK-Android
# Documentation: Remove release candidates documentation from the repo

Closes #428 

### Description

Remove the release candidates' documentation from the repo.

| Before | After |
|----------|-----------|
|<img width="111" alt="image" src="https://github.com/user-attachments/assets/37302eb8-e349-409f-86f4-78ae69580203">|<img width="68" alt="Screenshot 2024-11-06 at 10 47 32" src="https://github.com/user-attachments/assets/409ddbd3-7e75-420d-9e14-c9615d7bd471">|

Also, the `uiTestUtils` module is removed from the documentation. 

| Before | After |
|----------|-----------|
|<img width="350" alt="image" src="https://github.com/user-attachments/assets/bb76b2fa-0e35-448f-9897-e8aaa9e75ca1">|<img width="350" alt="image" src="https://github.com/user-attachments/assets/4554588c-fd95-4349-a578-ea35c9131302">|

### Testing Steps

- Review the documentation in the PR and check that the changes are there.
- Review the `build.gradle` files that have been modified.
